autoscheduler/user_sessions/views.py

```diff
@@ -97,6 +98,49 @@ def save_availabilities(request):
     """ Saves availabilities for the given user in the session """
     return _set_state_in_session(request, 'availabilities')
 
+@api_view(['GET'])
+def get_saved_schedules(request):
+    """ Returns the saved schedules from the session for the requested term """
+    term = request.query_params.get('term')
+
+    if not term:
+        return Response(status=400)
+
+    with retrieve_data_session(request) as data_session:
+        schedules = data_session.get(term, {}).get('schedules')
```

Comment:
Why doesn't this use `_get_state_from_session`

Reply:
Oh yeah duh - I knew it was going to be different from the normal `_get_state_from_session` but forgot I could just use it in the first part

Reply:
Oh wait it's b/c _get_state_from_session returns a `Response` and we need it as a list - I'll instead change `_get_state_from_session` to just return the object and each individual route will form it as a `Response`